### DISCUSSION TITLE
Move the Query Tab to the Top

### DISCUSSION THREAD

**Discussion by daryl-lynch-bzy** [DaxStudio/DaxStudio#913] [Q&A] [answered]
Hi, is it possible to move the Query Tabs back to the top?

![image](https://user-images.githubusercontent.com/76914701/187487907-d5a3def5-9a81-490d-8dff-d61820daf734.png)

Many thanks
Daryl

**Comment by marcosqlbi** [comment, marked as answer]
I know it takes some time to get used to the new UI, but the new design has been carefully considered, so we are not going to move it back.
